src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/ClinicalVariableCard.js

```diff
@@ -242,6 +243,7 @@ const ClinicalVariableCard = ({
         padding: '0.5rem 1rem 1rem',
         ...style,
       }}
+      key={key}
```

Comment:
this is what stops the card from re-mounting - cards are in a list, they need keys

Reply:
i don't think the `key` is getting passed all the way down from the card components in `ClinicalAnalysisResult` - i'm seeing `undefined` when i log it and the cards are still remounting.